Makefile

```diff
@@ -180,12 +180,17 @@ deps-quicktype-elm: verify-prereq-quicktype
 gen: gen-c gen-python gen-javascript gen-java gen-haskell gen-rust gen-protobuf gen-jsonschema gen-quicktype
 gen-quicktype: gen-quicktype-typescript gen-quicktype-elm
 
+gen-c_args = -i $(SBP_SPEC_DIR) \
+             -o $(SWIFTNAV_ROOT)/c \
+             -r $(SBP_VERSION)
+ifeq ($(SBP_STAGING), 1)
+	gen-c_args += -s
+endif
+
```

Comment:
Not the sure if it is the best approach to append a flag conditionally.

Reply:
Seems fine, this makes the behavior more explicit

Reply:
As an alternative I could see moving this behavior inside the generator an implicitly deciding if we're in a "staging" mode based on some environmental condition, but again, passing the switch makes the behavior more explicit